Library/Formula/links.rb

```diff
@@ -38,6 +35,6 @@ def install
   end
 
   test do
-    system "#{bin}/links", "https://duckduckgo.com"
+    system bin/"links", "https://duckduckgo.com"
```

Comment:
Sure. I got it in my mind that someone, somewhere said it looked cleaner this way, but it's possible I just imagined that. I'll stop changing them around :smiley_cat: